app/src/main/java/com/sdpteam/connectout/profile/RegisteredEventsDataSource.java

```diff
@@ -1,27 +1,13 @@
 package com.sdpteam.connectout.profile;
 
-import android.view.View;
-import android.widget.ListAdapter;
-
-import com.firebase.ui.database.FirebaseListOptions;
-
-import java.util.function.BiConsumer;
-import java.util.function.Consumer;
-import java.util.function.Function;
 
 public interface RegisteredEventsDataSource {
 
     /**
      * stores a new Profile.CalendarEvent (eventId, eventTitle and eventDate)

```

Comment:
I am allergic to this legacy code about Calendar.
Can you delete any mention of it in our code (do a Ctrl+Shift+F to search) so it does not confuse our team in the future.

Reply:
does it give you a rash? :)
I'm on it